jax/_src/lax/lax.py

```diff
@@ -2468,6 +2468,16 @@ def _named_call_translation_rule(c, axis_env, in_nodes, name_stack,
 
 xla.call_translations[named_call_p] = _named_call_translation_rule
 
+
+def named_call(f, name):
```

Comment:
Doc string? This is a new public symbol after all 😄 

We should also be adding this to https://github.com/google/jax/blob/master/docs/jax.lax.rst.

Reply:
Done.